pkg/apis/core/v1beta1/types_shoot.go

```diff
@@ -1061,6 +1066,13 @@ type KubeletConfig struct {
 	// This field is only available for Kubernetes v1.25 or later.
 	// +optional
 	SeccompDefault *bool `json:"seccompDefault,omitempty" protobuf:"varint,21,opt,name=seccompDefault"`
+	// A quantity defines the maximum size of the container log file before it is rotated. For example: "5Mi" or "256Ki".
+	// +optional
+	// Default: 100Mi
+	ContainerLogMaxSize *resource.Quantity `json:"containerLogMaxSize,omitempty" protobuf:"bytes,22,opt,name=containerLogMaxSize"`
+	// Maximum number of container log files that can be present for a container.
+	// +optional
+	ContainerLogMaxFiles *int32 `json:"containerLogMaxFiles,omitempty" protobuf:"bytes,23,opt,name=containerLogMaxFiles"`
```

Comment:
Please add these new fields to `example/90-shoot.yaml` file

Reply:
done